### ISSUE TITLE
Patched JDK makes font rendering worse

### ISSUE THREAD

**Issue by m-ober** [achaphiv/ppa-fonts#31]
Hi,

I'm using Linux Mint 18 Sarah. I'm also using the Xilinx Vivado IDE, which looks like this after installing Infinality. Note that the IDE comes with its own JRE.

![xilinx-jre](https://cloud.githubusercontent.com/assets/11992838/21149355/871b4c02-c15b-11e6-8174-08a6fc9cec99.png)

I installed the patched JRE and made the IDE use the patched JRE.

mober@mober-desktop ~ $ apt-cache policy openjdk-8-jre
openjdk-8-jre:
  Installed: 8u111-b14-2ubuntu0.16.04.2ppa1
  Candidate: 8u111-b14-2ubuntu0.16.04.2ppa1
  Version table:
 *** 8u111-b14-2ubuntu0.16.04.2ppa1 500
        500 http://ppa.launchpad.net/no1wantdthisname/openjdk-fontfix/ubuntu xenial/main amd64 Packages
        100 /var/lib/dpkg/status
     8u111-b14-2ubuntu0.16.04.2 500
        500 http://mirror.de.leaseweb.net/ubuntu xenial-updates/main amd64 Packages
        500 http://security.ubuntu.com/ubuntu xenial-security/main amd64 Packages
     8u77-b03-3ubuntu3 500
        500 http://mirror.de.leaseweb.net/ubuntu xenial/main amd64 Packages

lrwxrwxrwx 1 root root 64 Dec 13 17:33 /opt/Xilinx/SDx/2016.3/Vivado/tps/lnx64/jre/lib/amd64/server/libjvm.so -> /usr/lib/jvm/java-8-openjdk-amd64/jre/lib/amd64/server/libjvm.so

But - It's getting a lot worse:

![patched-jre](https://cloud.githubusercontent.com/assets/11992838/21149356/871d93b8-c15b-11e6-94dd-e4402eb80a7e.png)

Finally, I tried the tuxjdk (statically linked), which renders the fonts in a very nice way, but unfortunately a bit too  large.

![tux-jre](https://cloud.githubusercontent.com/assets/11992838/21149357/871dde7c-c15b-11e6-8d87-25a0064fa68a.png)

Is it possible to have the correct font size with "some" anti-aliasing? :smile:

**Comment by achaphiv** [comment]
Was this because of the recent update to the package in my ppa?

Or is this just the first time you're using it?

Because I just checked, and I'm using the [patch from the latest commit of tuxjdk](https://github.com/tuxjdk/tuxjdk/blob/058db08a5176102f1dc02b86fbf43727fd830ef3/quilt-patches/tuxjdk/add-fontconfig-support.diff).

So I don't know why it doesn't work.

Maybe try `-Dawt.useSystemAAFontSettings=lcd`?

**Comment by m-ober** [comment]
No, it also happened with the old version. I'm already using "-Dawt.useSystemAAFontSettings=lcd", but it does not seem to have any impact.
